### PR TITLE
Add new bodhi2 messages

### PR DESCRIPTION
**Pull request by ralphbean**
This is still a work in progress.

- [x] mashtask.start
- [x] mashtask.mashing
- [x] update.ejected
- [x] mashtask.complete
- [x] mashtask.sync.wait
- [x] mashtask.sync.done
- [x] update.complete.%s
- [x] update.comment
- [x] stack.save
- [x] stack.delete
- [x] update.edit
- [x] update.request.%s (these just worked without needing a change)
- [x] buildroot_override.tag
- [x] buildroot_override.untag

### CODE REVIEW

File: fedmsg_meta_fedora_infrastructure/bodhi.py [hunk 1]
```diff
@@ -121,12 +129,31 @@ def subtitle(self, msg, **config):
...
             author = msg['msg']['update']['submitter']
+            if isinstance(author, dict):
+                author = author['name']
             package = msg['msg']['update']['title']
             status = msg['msg']['update']['status']
             tmpl = self._(
                 "{author}'s {package} bodhi update completed push to {status}"
             )
             return tmpl.format(author=author, package=package, status=status)
+        elif 'bodhi.update.eject' in msg['topic']:
```
Comment (pypingou) [inline fedmsg_meta_fedora_infrastructure/bodhi.py]: eject vs reject?

**Comment by pypingou** [comment]
Travis and I are happy :+1: :smile:

File: fedmsg_meta_fedora_infrastructure/bodhi.py (hunk 1 above)
Comment (ralphbean) [inline fedmsg_meta_fedora_infrastructure/bodhi.py]: Yeah, ``eject``.  :)

The bodhi2 mash starts with a bunch of updates and as it goes along it figures out that updates X and Y are no good and so it "ejects" them.

Comment (pypingou) [inline fedmsg_meta_fedora_infrastructure/bodhi.py]: oh, cool, sounds nice :)

**Comment by ralphbean** [comment]
Oh, cool.  Thanks!

**Pull request merged**